batched/KokkosBatched_Util.hpp

```diff
@@ -21,6 +21,7 @@
 // no experimental name space guard for trilinos
 #define __KOKKOSBATCHED_PROMOTION__ 1
 
+#include <iostream>
```

Comment:
Should be taken care of so just need to rebase on develop

Reply:
Agree. I added this temporarily before the fixing PR was merged.